spec/system/provider_interface/see_individual_application_new_personal_statement_spec.rb

```diff
@@ -1,6 +1,5 @@
 require 'rails_helper'
 
-ANYTIME_IN_RECRUITMENT_CYCLE_APPLY_OPEN = Time.zone.local(2022, 10, 15, 12, 0, 0)
 RSpec.describe 'A Provider viewing an individual application', time: ANYTIME_IN_RECRUITMENT_CYCLE_APPLY_OPEN, with_audited: true do
```

Comment:
Have you looked at using `TimeTableHelper` `mid_cycle`

https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/spec/support/test_helpers/cycle_timetable_helper.rb#L12

Example:
https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/spec/rails_helper.rb#L97-L99

Reply:
I removed the constant and replaced it with `mid_cycle(2023)`. Thanks 👍